lesson_02/routes/app.route.js

```diff
@@ -0,0 +1,63 @@
+import { Router } from 'express';
+import { addUserSchema, updateUserSchema, removeUserSchema } from '../schemas/user.schema';
+import { getLoginMatch, handleError } from '../utils';
+
+export const router = Router();
+
+const store = {};
```

Comment:
it's not necessary to fix it now, but it's better to have a separate module which will have logic related to store

Reply:
Done